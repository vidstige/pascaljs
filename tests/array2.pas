program array2;
type
    TPoint = record
        x, y: Integer;
    end;
var
  a: array[1..2] of TPoint;
begin
  a[1].x := 2;
  a[1].y := 3;
  WriteLn('a[1].x = ', a[1].x, ', a[1].y = ', a[1].y);
end.