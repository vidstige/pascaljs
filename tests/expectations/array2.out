a[1].x = 2, a[1].y = 3
